Storage/ThirdParty/SwiftData.swift

```diff
@@ -267,6 +267,8 @@ private class SQLiteDBStatement {
             } else if obj is Date {
                 let timestamp = (obj as! Date).timeIntervalSince1970
                 status = sqlite3_bind_double(pointer, Int32(index+1), timestamp)
+            } else if obj is UInt64 {
```

Comment:
I ran into a similar issue when I was working on fixing another test case in Storage and it was actually being caused by an incorrect migration from NSNumber [1]. Could that also be the case here?

[1] https://github.com/mozilla-mobile/firefox-ios/commit/71bc3917d27f09a8806d279694368768933cd71f